src/client/rooms/room-page.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+
+export default class RoomPage extends Component {
+    render() {
+        return (
+            <div className="col-lg-4 text-center">
+            <h2>{`Room ${this.props.params.id}`}</h2>
```

Comment:
This'll look weird for most cool names for a room, 
`Room the friday room`
`Room tom's room`
`Room library`

Can we get a colon at least `Room: the commit room`